### ISSUE TITLE
Hello, author, I like this app very much, but I am a beginner and have some questions I would like to ask you

### ISSUE THREAD

**Issue by chang7ing** [zcemycl/Matlab-GAN#10]
Can A CGAN be used to generate images for data? What is the specific name of the generated neural network? How do you save a trained network?

**Comment by zcemycl** [comment]
@chang7ing 
Have done that code quite a while ago, so it may not be true. 
The model is customized before, it uses externally defined weights, instead of weights living inside the object. So if you want to use the model, please use functions (at the bottom of the scripts) like Generator and Discriminator by passing inputs, parameters (paramGen paramDis for cnn/fnn layers) and states (stGen stDis for batch norm).
You can save the model by saving the parameters and states. There are functions (like save) that can save them into mat files. Afterwards, you can load them, and pass them back to the functions (at the bottom of all scripts).

**Comment by chang7ing** [comment]
您好，我已收到！Thank you for your answer sheet. I have received your answer sheet.

**Comment by woshixiaohanpi** [comment]
您好！我在做毕业设计，您的代码我想认真学习一下，但是好多地方我还是看不懂，请问有没有这个代码的详细解析呢？我想看一下，谢谢您！

**Comment by zcemycl** [comment]
@woshixiaohanpi Suggestions: 
1. implement gan with matlab example (not mine).
2. understand the custom training loop.
3. read my gan script, and compare it with matlab one. 
4. read keras/pytorch implementation. Understand how it differs from matlab implementation. 
5. follow the sequence of when each gan model is created, don't skip. I created gan first, then dcgan, then cgan, then infogan, then pix2pix, etc. If you skip, you might not understand my syntax. In later model, like cyclegan or pix2pix, I use exec a lot to reduce repetitive code. 

These scripts are created within 1 month, I don't do a lot of comments, and not intended to since I have no more matlab :).
